.github/workflows/linux_simd.yml

```diff
@@ -224,10 +220,6 @@ jobs:
 
     - name: Install dependencies
       run: |
-        sudo apt update
-        sudo apt install -y g++-13
-        sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-13 1
-        sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-13 1
```

Comment:
Perhaps explicitly set `CXX=g++-13` and `CC=gcc-13` in the `Build` section below for my sanity? :)